### DISCUSSION TITLE
Can't download any Songs

### DISCUSSION THREAD

**Discussion by Frikandelchris** [vicwomg/pikaraoke#248] [General]
Hello people,

I installed PiKaraoke. The Installation process worked for me and i have a connection to the internet, but i can't download any songs. I Already did an update for "youtube-dl", but it still doesnt work. (version 2022.04.08).

I can search and find songs on youtube, "start" the download - but then nothing happens.

How can i fix that?

**Comment by vicwomg** [comment]
That version of yt-dlp is too old. Pull the latest pikaraoke  ( cd to the pikaraoke directory, run `git pull` ), relaunch pikaraoke, and then go to the web interface and pikaraoke info section, then click the link to update yt-dlp

The reason you need to git pull is the current image you're running likely has a version of pikaraoke that has a permissions bug in the yt-dlp updater which should be fixed in #238

**Comment by Frikandelchris** [comment]
`sudo pip3 install --upgrade yt-dlp` worked for me.

`git pull` didn't work

**Comment by vicwomg** [comment]
After you ran git pull did you go into pikaraoke's interface and try the yt-dlp upgrade from there? Running git pull alone doesn't do it.
